modules/afmongodb/afmongodb.c

```diff
@@ -144,7 +144,7 @@ _format_persist_name(const LogPipe *s)
 static inline void
 _uri_set_default_server_selection_timeout(mongoc_uri_t *uri_obj)
 {
-#if SYSLOG_NG_HAVE_DECL_MONGOC_URI_SET_OPTION_AS_INT32
+#if SYSLOG_NG_HAVE_DECL_MONGOC_URI_SET_OPTION_AS_INT32 && SYSLOG_NG_HAVE_DECL_MONGOC_URI_SERVERSELECTIONTIMEOUTMS
```

Comment:
Do we have to check for both `MONGOC_URI_SERVERSELECTIONTIMEOUTMS` and `mongoc_uri_set_option_as_int32` ? Maybe it is guaranteed that if `MONGOC_URI_SERVERSELECTIONTIMEOUTMS` the `mongoc_uri_set_option_as_int32` also exists ?